### ISSUE TITLE
Compilation error on 1.1.1+1

### ISSUE THREAD

**Issue by Zazo032** [kushalmahapatro/cuberto_bottom_bar#12]
Error log:
```
Compiler message:                                                       
../.pub-cache/hosted/pub.dartlang.org/cuberto_bottom_bar-1.1.1+1/lib/cuberto_bottom_bar.dart:4:8: Error: Error when reading '../.pub-cache/hosted/pub.dartlang.org/cuberto_bottom_bar-1.1.1+1/lib/internal/tab_item_new.dart': No such file or directory
import 'package:cuberto_bottom_bar/internal/tab_item_new.dart';         
       ^                                                                
Target kernel_snapshot failed: Exception: Errors during snapshot creation: null
build failed. 
```

**Comment by kushalmahapatro** [comment]
@Zazo032 Fixed this , please check 1.1.1+2

**Comment by Zazo032** [comment]
It's working properly with `1.1.1+2`. Thanks!
